Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift

```diff
@@ -1,8 +1,9 @@
 import UIKit
 
 class ImageContainerView: UIView {
+    private static let animationLength: TimeInterval = 0.25
```

Comment:
This I would love to be in Constants and make sure all our animation durations are consistent